### DISCUSSION TITLE
JWT解析器 payload  和 header 显示框 无法拉长

### DISCUSSION THREAD

**Discussion by cuibty** [he3-app/he3-feedback#33] [General]
![Uploading image.png…]()

**Comment by hongchaodeng** [comment]
你好，看不到图片哦。麻烦重新上传一下~

**Comment by cuibty** [comment]
![image](https://user-images.githubusercontent.com/12811343/223027415-10a746d5-ab81-4dd0-8d84-d1301c9f3299.png)
@hongchaodeng
